WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailCommentsTableViewDelegate.swift

```diff
@@ -50,8 +60,18 @@ class ReaderDetailCommentsTableViewDelegate: NSObject, UITableViewDataSource, UI
             return nil
         }
 
-        let titleFormat = totalComments == 1 ? Constants.singularCommentFormat : Constants.pluralCommentsFormat
-        header.titleLabel.text = String(format: titleFormat, totalComments)
+        let title: String = {
+            switch totalComments {
+            case 0:
+                return Constants.comments
+            case 1:
+                return String(format: Constants.singularCommentFormat, totalComments)
+            default:
+                return String(format: Constants.pluralCommentsFormat, totalComments)
+            }
+        }()
+
+        header.titleLabel.text = title
```

Comment:
Just a _very_ minor nitpick, but if there's no reason behind creating the `title` variable, the closure could be directly assigned here.
```
header.titleLabel.text = { 
    ... 
}()
```